pkgs/development/python-modules/albumentations/default.nix

```diff
@@ -0,0 +1,46 @@
+{ fetchPypi
+, buildPythonPackage
+, lib
+, numpy
+, scipy
+, scikitimage
+, pyyaml
+, qudida
+, pytorch
+, torchvision
+, imgaug
+, pytest
+, opencv3
+}:
+
+buildPythonPackage rec {
+  pname = "albumentations";
+  version = "1.1.0";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "YLBnswk5CLzFKtsqpdRPV+vbuKtXpHsLQvPcHTsc6CQ=";
+  };
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    scikitimage
+    pyyaml
+    qudida
+  ];
+
+  checkInputs = [ pytest pytorch torchvision imgaug opencv3 ];
```

Comment:
Normally I would suggest switching to `pytestCheckHook` as mentioned below, which would also allow removing optional dependencies since the test runner skips the relevant tests automatically, but it seems to hang on this test suite for some reason.